ansible/roles/test/files/ptftests/vnet_vxlan.py

```diff
@@ -300,11 +301,21 @@ def setUp(self):
         self.generate_ArpResponderConfig()
 
         self.cmd(["supervisorctl", "start", "arp_responder"])
-
+        self.check_arp_responder_running()
         self.dataplane.flush()
 
         return
 
+    def check_arp_responder_running(self):
```

Comment:
what if the arp responder is not up after 5 seconds - in this case I don't see any exception thrown.

Have you considered using the retry library or retry decorator?

Reply:
retry library or decorator is not installed to ptf docker.

Reply:
I remember there was an recent change to install retry in sonic-mgmt docker. But we do need to be careful here to make sure people using older sonic-mgmt docker would have a fair warning.